stdlib/universe/state_tracking.go

```diff
@@ -154,7 +156,7 @@ func createStateTrackingTransformation(id execute.DatasetID, mode execute.Accumu
 	}
 	cache := execute.NewTableBuilderCache(a.Allocator())
 	d := execute.NewDataset(id, mode, cache)
-	t, err := NewStateTrackingTransformation(d, cache, s)
+	t, err := NewStateTrackingTransformation(d, cache, s, a.Context(), a.Dependencies()["interpreter"].(dependencies.Interface))
```

Comment:
we should be able to change this to executetest.NewTestExecuteDependencies()

Reply:
not a test.